api/config/initializers/koala.rb

```diff
@@ -0,0 +1,5 @@
+Koala.configure do |config|
```

Comment:
I haven't used this gem before, seems great for what you need.  My normal pick for auth is devise.  If i only need oauth i generally use omniauth

Reply:
cool - I was looking at both devise and oauth, but opted for something much simpler for now since I will be handling the oauth portions on the client and then using the token returned to authenticate with this graphql api. I might add in devise later on if I start handling accounts on the api itself